service/src/test/java/com/epam/rft/atsy/service/passwordchange/validation/impl/PasswordOldPasswordMatchesRuleTest.java

```diff
@@ -4,97 +4,86 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 import static org.mockito.BDDMockito.given;
-import static org.mockito.BDDMockito.then;
-import static org.mockito.Mockito.verifyNoMoreInteractions;
-import static org.mockito.Mockito.verifyZeroInteractions;
 
 import com.epam.rft.atsy.service.domain.PasswordChangeDTO;
+import com.epam.rft.atsy.service.security.CustomSecurityContextHolderService;
+import com.epam.rft.atsy.service.security.UserDetailsAdapter;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.runners.MockitoJUnitRunner;
-import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
-import org.springframework.security.core.Authentication;
-import org.springframework.security.core.context.SecurityContextHolder;
-import org.springframework.security.core.userdetails.UserDetails;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 
 @RunWith(MockitoJUnitRunner.class)
 public class PasswordOldPasswordMatchesRuleTest {
 
+  private static final Long USER_ID = 1L;
+  private static final String USER_NAME = "Test";
+
   private static final String NEW_PASSWORD = "new password";
   private static final String NEW_PASSWORD_CONFIRM = NEW_PASSWORD;
-  private static final String OLD_PASSWORD = "password as hash";
   private static final String PASSWORD_CHANGE_DTO_OLD_PASSWORD = "password";
   private static final String PASSWORD_CHANGE_DTO_DIFFERENT_OLD_PASSWORD = "differentpassword";
   private static final String MESSAGE_KEY = "passwordchange.validation.oldpasswordmatch";
 
-  @Mock
-  private BCryptPasswordEncoder bCryptPasswordEncoder;

```

Comment:
In my view the BCryptPasswordEncoder should be hidden from the controllers and services, however now it is okay. We will do it in another task.